apps/api/e2e/tests/trading.test.ts

```diff
@@ -1532,4 +1537,61 @@ describe("Trading API", () => {
       "EVM-to-SVM trade failed as expected with disallowXParent setting",
     );
   });
+
+  test("small numbers can be inserted into database", async () => {
+    // Setup admin client
+    const adminClient = createTestClient();
+    await adminClient.loginAsAdmin(adminApiKey);
+
+    // Register team and get client
+    const { team } = await registerTeamAndGetClient(
+      adminApiKey,
+      "Reason Required Team",
+    );
+
+    // Start a competition with our team
+    const competitionName = `Reason Required Test ${Date.now()}`;
+    const competition = await startTestCompetition(
+      adminClient,
+      competitionName,
+      [team.id],
+    );
+
+    // Wait for balances to be properly initialized
+    await wait(500);
+
+    const smallValue = 2.938e-27;
+
+    const trade: InsertTrade = {
```

Comment:
These values a a bit arbitrary, but the important thing is that the number fields are all set to the value that caused the issue during the last comp